std/node/_fs/_fs_stat_test.ts

```diff
@@ -0,0 +1,107 @@
+import { BigIntStats, stat, Stats, statSync } from "./_fs_stat.ts";
+import { assertEquals, fail } from "../../testing/asserts.ts";
+
+export function assertStats(actual: Stats, expected: Deno.FileInfo) {
+  assertEquals(actual.dev, expected.dev);
+  assertEquals(actual.gid, expected.gid);
+  assertEquals(actual.size, expected.size);
+  assertEquals(actual.blksize, expected.blksize);
+  assertEquals(actual.blocks, expected.blocks);
+  assertEquals(actual.ino, expected.ino);
+  assertEquals(actual.gid, expected.gid);
+  assertEquals(actual.mode, expected.mode);
+  assertEquals(actual.nlink, expected.nlink);
+  assertEquals(actual.rdev, expected.rdev);
+  assertEquals(actual.uid, expected.uid);
+  assertEquals(actual.atime?.getTime(), expected.atime?.getTime());
+  assertEquals(actual.mtime?.getTime(), expected.mtime?.getTime());
+  assertEquals(actual.birthtime?.getTime(), expected.birthtime?.getTime());
+  assertEquals(actual.atimeMs, expected.atime?.getTime());
+  assertEquals(actual.mtimeMs, expected.mtime?.getTime());
+  assertEquals(actual.birthtimeMs, expected.birthtime?.getTime());
+  assertEquals(actual.isFile(), expected.isFile);
+  assertEquals(actual.isDirectory(), expected.isDirectory);
+  assertEquals(actual.isSymbolicLink(), expected.isSymlink);
+  // assertEquals(actual.ctimeMs === expected.ctime?.getTime());
+  // assertEquals(actual.ctime?.getTime() === expected.ctime?.getTime());
```

Comment:
Remove?

Reply:
Deno doesn't provide a "ctime" in stat and lstat so, i used "mtime" for "ctime"

Reply:
> Remove?

yes